src/feature-flags.ts

```diff
@@ -58,6 +59,11 @@ export const featureConfig: Record<
   Feature,
   { envVar: string; minimumVersion: string | undefined; defaultValue: boolean }
 > = {
+  [Feature.CodeqlJavaLombokEnabled]: {
+    envVar: "CODEQL_JAVA_LOMBOK_ENABLED",
```

Comment:
For consistency with the others environment variable overrides for the feature flag! 
```suggestion
    envVar: "CODEQL_JAVA_LOMBOK",
```